File: api/hospital-system/src/main/java/com/api/hospitalsystem/model/maritalStatus/MaritalStatus.java
package com.api.hospitalsystem.model.maritalStatus;

public enum MaritalStatus {

    Married("Casado"),
    StableUnion("União Estável"),
    Widower("Viúvo"),
    Single("Solteiro");

    private final String description;

    MaritalStatus(String description) {
        this.description = description;
    }

    public String getValue() {
        return description;
    }

    @Override
    public String toString() {
        return description;
    }


}

File: app/hospital-system/src/app/module/models/src/marital-status/marital-status.ts
import { describe } from '@module/utils/functions/enum';

export enum MaritalStatus {
  Married = 0,
  StableUnion = 1,
  Widower = 2,
  Single = 3,
}

describe(MaritalStatus, {
  Married: 'Casado',
  StableUnion: 'União Estável',
  Widower: 'Viúvo',
  Single: 'Solteiro',
});
